tools/sign.bat

```diff
@@ -0,0 +1,15 @@
+@echo off
+
+set timeservers=(http://timestamp.globalsign.com/scripts/timestamp.dll http://timestamp.comodoca.com/authenticode http://timestamp.verisign.com/scripts/timestamp.dll http://tsa.starfieldtech.com)
+
+for %%s in %timeservers% do (
+    signtool sign /a /d "Node.js" /du "https://nodejs.org" /t %%s %1
+    if ERRORLEVEL 0 if not ERRORLEVEL 1 (

```

Comment:
This is just a nit at this point, but `if ERRORLEVEL 0 ` is redundant here. (Ref: http://ss64.com/nt/errorlevel.html )

Also, according to https://msdn.microsoft.com/en-us/library/8s9b9yaz(v=vs.110)#Anchor_6 , perhaps we should also accept exit code 2. But we never did and it's not clear what kind of warnings might happen, so let's just be watchful for binaries signed twice or all servers failing with a different error in the first one.